src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -236,11 +243,21 @@ private function formatTime(float $time): float
         return round($time, 1);
     }
 
+    /**
+     * Gets the event name.
+     *
+     * @return string The event name
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
     /**
      * @return string
      */
     public function __toString()
     {
-        return sprintf('%s: %.2F MiB - %d ms', $this->getCategory(), $this->getMemory() / 1024 / 1024, $this->getDuration());
+        return sprintf('%s: %.2F MiB - %d ms', ($this->getName() ?? $this->getCategory()), $this->getMemory() / 1024 / 1024, $this->getDuration());
```

Comment:
I think this should rather include both the name and category rather the hiding the category when there is a name

Reply:
I thought about it as well. I would say we go for hiding the `category` and only show the name!

Reply:
well, my suggestion is that non-default categories should not be hidden (and then, there is a question about what happens for empty name indeed, where I would probably keep the behavior of showing the category even when it is the default one)

Reply:
Yes, what do you think about this:
```php
$stopwatch = new Stopwatch();
$stopwatch->start('foo', 'cat');
$event = $stopwatch->stop('foo');
echo (string) $event;
```
... gives: `foo in category cat: 2.00 MiB - 0 ms`

and:
```php
$stopwatch = new Stopwatch();
$stopwatch->start('foo');
$event = $stopwatch->stop('foo');
echo (string) $event;
```
... gives:`foo in category default: 2.00 MiB - 0 ms`.

We will always have a name but the category might have it and might not so I thought about introducing this new format!

Reply:
I would use something more compact like: `cat/foo: ...`, `default/foo: ...`